Library/Core/UnoCore/Targets/Android/app/src/main/res/values/styles.xml

```diff
@@ -5,7 +5,11 @@
 #if !@(Project.Mobile.ShowStatusbar)
         <item name="android:windowFullscreen">true</item>
 #endif
-        <item name="android:windowIsTranslucent">true</item>
+#if !@(Project.Android.WindowIsTranslucent)
+        <item name="android:windowIsTranslucent">@(Project.Android.WindowIsTranslucent)</item>
+#else
+        <item name="android:windowIsTranslucent">false</item>
+#endif
```

Comment:
Can't you just do this?
```xml
<item name="android:windowIsTranslucent">@(Project.Android.WindowIsTranslucent || @"false")</item>
```

Reply:
Good idea! Done!

Reply:
It's all done. But still showing `Changes requested`. i need to do anithing more?

Thanks!